app/controllers/people_controller.rb

```diff
@@ -0,0 +1,17 @@
+class PeopleController < ApplicationController
+  def index
+    @all_people = Person.order(id: :asc)
+    render :index
+  end
+
+  def show
+    @person = Person.find(params[:id])
+    @tasks = @person.tasks
+    render :show
+  end
+
```

Comment:
If `person` is being used just for `@tasks` you could make it a local variable, but if it's being used in the view this is appropriate.